<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventDisplay</class>
 <widget class="QWidget" name="EventDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>436</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>436</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shows the chances for the type of the next question mark floor. I have done absolutely zero testing of the accuracy of these numbers and am currently just pulling them directly from the save file. [The problem: the save file doesn't always report the proper values; e.g. the potion chance is saved as 40 percent less than it would be otherwise -- I haven't researched these numbers enough to verify if they're correct]&lt;/p&gt;&lt;p&gt;Once I do the legwork to figure exactly what's going on with these values, I'll hopefully have a more detailed writeup and replace this tool tip as necessary.&lt;br/&gt;&lt;/p&gt;&lt;p&gt;From top to bottom:&lt;/p&gt;&lt;p&gt;* Regular Events&lt;/p&gt;&lt;p&gt;* Normal Fight&lt;/p&gt;&lt;p&gt;* Shop&lt;/p&gt;&lt;p&gt;* Treasure Chest&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources/resources.qrc">:/icons/event.svg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>188</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>188</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_event_chance">
     <property name="font">
      <font>
       <family>Kreon</family>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string>40%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_monster_chance">
     <property name="font">
      <font>
       <family>Kreon</family>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string>10%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>188</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources/resources.qrc">:/icons/monster.svg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_shop_chance">
     <property name="font">
      <font>
       <family>Kreon</family>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources/resources.qrc">:/icons/shop.svg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources/resources.qrc">:/icons/chest.svg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>188</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_chest_chance">
     <property name="font">
      <font>
       <family>Kreon</family>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
